egs/swbd/s5c/local/nnet3/tuning/run_tdnn_3c.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# 3c is as a, but uses xconfig.
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+# System                   tdnn_a   tdnn_3c
+# WER on train_dev(tg)      17.41     17.37
```

Comment:
oh... you didn't show the train_dev numbers before... this seems to confirm that what we saw on eval2000 was just noise.